keywords/TestServeriOS.py

```diff
@@ -29,7 +30,15 @@ def __init__(self, version_build, host, port, community_enabled=None, debug_mode
         self.app_name = ""
         self.platform = platform
         self.bundle_id = ""
-
+        self.released_version = {
+            "2.0.0": 806,
+            "2.1.0": 263,
+            "2.1.1": 10,
+            "2.1.2": 11,
+        }
+
+        if self.version_build == "2.1.0":
```

Comment:
do you mean < "2.1.0" or == is intentional. If == is intentional, then ios build available  which are below "2.1.0" builds?

Reply:
app is not available for this version, hence ==